src/Elasticsearch.Net/ConnectionPool/SniffingSortedStickyConnectionPool.cs

```diff
@@ -5,74 +5,31 @@
 
 namespace Elasticsearch.Net
 {
-	public class SniffingSortedStickyConnectionPool : IConnectionPool
+	public class SniffingSortedStickyConnectionPool : SniffingConnectionPool
```

Comment:
👍 I like this :) Maybe also refactor the common bits of `CreateView` to protected virtual members of Sniffing/Static connection pool?